src/pages/ProductList/ProductList.js

```diff
@@ -1,16 +1,23 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { BiUpArrow, BiDownArrow, BiFilter } from 'react-icons/bi';
 import Dropdown from './Product/Dropdown';
+import ProductAside from './ProductAside';
 import './ProductList.scss';
 
 const ProductList = () => {
   const [dropdownShown, setDropdownShown] = useState(false);
 
-  const [data, setData] = useState([]);
+  const [productMain, setProductMain] = useState([]);
 
-  fetch('/data/mock.json')
-    .then(response => response.json())
-    .then(data => setData(data));
+  useEffect(() => {
+    fetch('http://10.58.52.118:3000/main?offset=0&limit=10', {
+      method: 'GET',
+    })
+      .then(response => response.json())
+      .then(item => setProductMain(item));
```

Comment:
```suggestion
      .then(setProductMain);
```

들어오는 인자와 콜백함수에서 받는 인자 값이 동일하면 위와 같이 축약형으로도 사용할 수 있습니다!